Gemfile

```diff
@@ -49,6 +49,7 @@ gem 'shotgun',   '~> 0.9', require: false
 
 gem 'hanami-devtools', require: false, git: 'https://github.com/hanami/devtools.git'
 gem 'hanami-webconsole', require: false, git: 'https://github.com/hanami/webconsole.git'
+gem 'semantic_logger'
```

Comment:
@davydovanton Can you please remove this dependency? A custom, non-working logger can be mocked like this:

```ruby
class SemanticLogger
  def initialize(*)
  end

  def info(*)
  end

  def warn(*)
  end

  def debug(*)
  end
end
```

Reply:
Awesome, will fix it today